echidna/calc/decay.py

```diff
@@ -0,0 +1,89 @@
+import numpy
+
+_avagadros = 6.02214129e23  # PDG Avagadro's constant
+_electron_mass = 0.510998910e6  # PDG eV/c^2
+
+
+def get_n_atoms(mass, atm_weight, abundance):
+    """ Calculates the number of atoms of an isotope.
+    Args:
+      mass (float): Mass of isotope in kg.
+      atm_weight (float): Atomic weight of isotope in g/mol.
+      abundance (float): Natural abundance of isotope with 0 to 1 equivalent
+      to 0% to 100%.
+
+    Raises:
+      ValueError: If abundance is < 0. or > 1.
+
+    Returns:
+      float: Number of atoms.
+    """
+    if abundance < 0. or abundance > 1.:
+        raise ValueError("Abundance ranges from 0 to 1")
+    return (mass*1000.*_avagadros*abundance)/atm_weight
+
+
+def get_activity(halflife, n_atoms):
```

Comment:
We should probably agree on some consistent variable name for half-life throughout echidna, I know @arushanova's code currently uses `T`